lib/manager/components/version/FeedVersionViewer.js

```diff
@@ -218,37 +221,67 @@ class VersionSectionSelector extends Component<SelectorProps> {
       : <span><BsLabel bsStyle={color}>{text}</BsLabel></span>
   }
 
+  // Render the number of transformations applied to the current feed version
+  _renderTransformResultsCount = (version: FeedVersion) => {
+    let totalCount
+    if (version.feedTransformResult) {
+      const {tableTransformResults} = version.feedTransformResult
+      totalCount = tableTransformResults.reduce((prev, cur) => {
+        return cur.deletedCount + cur.updatedCount + cur.addedCount + prev
+      }, 0)
+    } else {
+      totalCount = 0
+    }
+    return (
+      <BsLabel>
+        {totalCount}
+      </BsLabel>
+    )
+  }
+
   render () {
     const { feedVersionIndex, version, versionSection } = this.props
     const versionPath = `/feed/${version.feedSource.id}/version/${feedVersionIndex}`
     return (
       <Panel>
         <ListGroup fill>
           <LinkContainer
+            active={!versionSection}
             to={versionPath}
-            active={!versionSection}>
+          >
             <ListGroupItem><Icon type='info-circle' /> Version summary</ListGroupItem>
           </LinkContainer>
           <LinkContainer
+            active={versionSection === 'issues'}
             to={`${versionPath}/issues`}
-            active={versionSection === 'issues'}>
+          >
             <ListGroupItem>
               <Icon type='exclamation-triangle' /> Validation issues {this._renderIssuesLabel()}
             </ListGroupItem>
           </LinkContainer>
+          <LinkContainer
+            active={versionSection === 'transformations'}
+            to={`${versionPath}/transformations`}
+          >
+            <ListGroupItem>
+              <Icon type='exclamation-triangle' /> Transformation Results {this._renderTransformResultsCount(version)}
```

Comment:
Why is it an `exclamation-triangle`? What do you think about using https://fontawesome.com/v4.7/icon/exchange?

Reply:
Oh my gosh, this is so obvious. I copied this and never changed it, despite seeing it every time I click on it. One of those things that after enough time you stop seeing, I guess.